### ISSUE TITLE
List of repositories is incomplete or not working correctly

### ISSUE THREAD

**Issue by DenisVuyka** [cla-assistant/cla-assistant#307]
URL to the linked Repo/Org:

Steps to reproduce the problem:

I have an admin rights to a project, that is part of the Organisation.

When signing in to CLA Assistant, I can see all my private repos, and several repositories from the Organisation, but not the ones I'm an admin of. Note that I'm not the Organisation admin.

It is not possible to configure the project.

What is the expected result?

It should be possible to select the repository from the select list.

What happens instead?

It shows some irrelevant repositories, but not those expected.

**Comment by KharitonOff** [comment]
@DenisVuyka, it sounds strange... are the repos, you are trying to link, private or public?
We do not support linkage of private repositories. 
You should actually see all repos from the org, where you are a member and you have push rights. (see https://github.com/cla-assistant/cla-assistant/issues/130#issuecomment-232600296)

**Comment by DenisVuyka** [comment]
@KharitonOff Absolutely public open source project: https://github.com/Alfresco/Alfresco-Content-App

**Comment by wabson** [comment]
We discovered the source of this issue, that myself and @DenisVuyka had just been added as admins via the Collaborators list, not via a team. So although we had the right permissions, the API only returns repositories where you are the owner or added via an org (or group within an org) and therefore it did not show up.

After adding a new group inside the org with ourselves in and then adding this group with admin permissions on the project, the repository could be selected.
